scripts/ebpf_tracing.cmd

```diff
@@ -0,0 +1,106 @@
+@rem Copyright (c) Microsoft Corporation
+@rem SPDX-License-Identifier: MIT
+
+@rem Script behavior:
```

Comment:
> Actually - these do not seem to be input parameters. Do we want them to be optional input parameters instead of hardcoded values?

We could, but then if something needs to be re-tuned, we will need to change the installation scripts of the caller, instead of simply changing the delivered script.

Reply:
Maybe we have a separate issue to parameterize the script.

Reply:
Added #2372